content/en/blog/_posts/2023-12-05-kubernetes-1.29.md

```diff
@@ -0,0 +1,172 @@
+    
+To address this problem, a fourth access mode ReadWriteOncePod was introduced as an Alpha feature in v1.22 for CSI volumes. If you create a pod with a PVC that uses the ReadWriteOncePod access mode, Kubernetes ensures that pod is the only pod across your whole cluster that can read that PVC or write to it. In v1.29, this feature became Generally Available.
+
+### Generally Available: Node Expand Secret for CSI Driver
+    
+In Kubernetes, a volume expansion operation may include the expansion of the volume on the node, which involves filesystem resize. Some CSI drivers require secrets, for example a credential for accessing a SAN fabric, during the node expansion for the following use cases:
+	•	When a PersistentVolume represents encrypted block storage, for example using LUKS, you need to provide a passphrase in order to expand the device.
+	•	For various validations, the CSI driver needs to have credentials to communicate with the backend storage system at time of node expansion. 
+
+To meet this requirement, a CSI Node Expand Secret feature was introduced in Kubernetes 1.25. This allows an optional secret field to be sent as part of the NodeExpandVolumeRequest by the CSI drivers so that node volume expansion operation can be performed with the underlying storage system. In Kubernetes 1.29, this feature became generally available.
+
+## What's New in [sig-scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling)
+
+### Beta: Per-plugin callback functions for accurate requeueing in kube-schedule
+
+The throughput of the scheduler is our eternal challenge. This QueueingHint feature brings a new possibility to optimize the efficiency of requeueing, which could reduce useless scheduling retries significantly.
+    
+### Alpha: Introduce MatchLabelKeys to Pod Affinity and Pod Anti Affinity - alpha in v1.29
+
+The throughput of the scheduler is our eternal challenge. This QueueingHint feature brings a new possibility to optimize the efficiency of requeueing, which could reduce useless scheduling retries significantly. We're actively working on this, and some stable in-tree plugins will have supported it in v1.29.
+    
+### Generally Available: ReadWriteOncePod access mode for PV
+
+### Beta: Node lifecycle separated from taint management
+
+As title describes, it's to decouple `TaintManager` that performs taint-based pod eviction from `NodeLifecycleController` and make them two separate controllers: `NodeLifecycleController` to add taints to unhealthy nodes and `TaintManager` to perform pod deletion on nodes tainted with NoExecute effect.
+
+## What's New in [sig-network](https://github.com/kubernetes/community/tree/master/sig-network)
+
+### Alpha: nftables backend for kube-proxy
+    
+This features adds a new backend to kube-proxy based on nftables, since kernel community is focused mainly in nftables and practically stopped the iptables development
```

Comment:
```suggestion
The default kube-proxy implementation on Linux is currently based on iptables. IPTables was the preferred packet filtering and processing system in the Linux kernel for many years (starting with the 2.4 kernel in 2001). However, unsolable problems with iptables led to the development of a successor, nftables. Development on iptables has mostly stopped, with new features and performance improvements primarily going into nftables instead.

This feature adds a new backend to kube-proxy based on nftables, since some Linuxndistributions already started to deprecate and remove iptables, and nftables claims to solve the main performance problems of iptables.
```